synthesizer/src/vm/helpers/sequential_op.rs

```diff
@@ -37,7 +44,17 @@ impl<N: Network, C: ConsensusStorage<N>> VM<N, C> {
                 // Perform the queued operation.
                 let ret = match op {
                     SequentialOperation::AddNextBlock(block) => {
+                        #[cfg(feature = "announce-blocks")]
+                        let ipc_payload = (block.height(), bincode::serialize(&block).unwrap()); // Infallible.
                         let ret = vm.add_next_block_inner(block);
+                        #[cfg(feature = "announce-blocks")]
+                        if ret.is_ok() {
+                            if let Err(e) = announce_block(&mut stream, ipc_payload) {
```

Comment:
Could blocks be skipped in this case if it takes longer than an iteration to restart the stream?